backend/routes/auth.py

```diff
@@ -44,3 +44,23 @@ def manual_register():
 
     except Exception as e:
         return Response(status=500, response=json.dumps({"error": "Internal server error", "details": str(e)}))
+
+    
+@auth_blueprint.route("/login", methods=["POST"])
+def manual_login():
+    try:
+        email = request.form['email']
+        password = request.form['password']
+    
+        if find_user_by_email(email):
+            if verify_password(email, password):
+                return Response("Login Successful", status=200, mimetype='application/json')
+            else:
+                return Response("Login Failed", status=401, mimetype='application/json')
+        else:
+            return Response("Login Failed", status=401, mimetype='application/json')
```

Comment:
**_MANDATORY:_**
- Throw errors for each condition. 
- At the end, send a Response OK